### DISCUSSION TITLE
Actions - Details do not show the date of the action

### DISCUSSION THREAD

**Discussion by gnglaudell** [community/community#76382] [Actions]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

As a GitHub user, When I look at Actions, I want to see the date time of the action, so that I can understand when it happened.

![Screen Shot 2023-11-20 at 9 41 32 PM](https://github.com/community/community/assets/31864774/53194523-b313-42b5-8711-9a9b63535430)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @gnglaudell ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
